android/java/org/chromium/chrome/browser/app/domain/CryptoModel.java

```diff
@@ -290,6 +335,11 @@ public void setAccountInfosFromKeyRingModel(
         mNetworkModel.setAccountInfosFromKeyRingModel(accountInfosFromKeyRingModel);
     }
 
+    // clear buy send swap model
```

Comment:
for the future, all comments should start with a capital letter:
`clear buy send swap model`